libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/signboard/model/MapboxExternalFileResolver.kt

```diff
@@ -0,0 +1,145 @@
+package com.mapbox.navigation.ui.maps.guidance.signboard.model
+
+import android.content.res.AssetManager
+import android.graphics.Bitmap
+import android.graphics.Typeface
+import com.caverock.androidsvg.SVGExternalFileResolver
+import java.lang.RuntimeException
+
+/**
+ * Default implementation of [SVGExternalFileResolver] class used by SVG Renderer to resolve fonts
+ * when rendering the SVG.
+ *
+ * Extend this class and override the methods if you want to customize how AndroidSVG treats fonts,
+ * image and external CSS references.
+ *
+ * Example usage:
+ *
+ * class MyResolver(assetManager: AssetManager): MapboxExternalFileResolver(assetManager) {
+ *     // Override the default method implementations with your own.
+ * }
+ */
+open class MapboxExternalFileResolver(
```

Comment:
Should it be? 🤔 

```suggestion
open class ExternalFileResolver(
```

Reply:
I think it's good to add `Mapbox` because we have a `SVGExternalFileResolver` as well.